src/opm/output/eclipse/Summary.cpp

```diff
@@ -1724,6 +1724,69 @@ namespace Evaluator {
         std::string saveKey_;
     };
 
+    class Day : public Base
+    {
+    public:
+        explicit Day(std::string saveKey)
+            : saveKey_(std::move(saveKey))
+        {}
+
+        void update(const std::size_t          sim_step,
+                    const double            /* stepSize*/,
+                    const InputData&           input,
+                    const SimulatorResults& /* simRes */,
+                    Opm::SummaryState&         st) const override
+        {
+            auto sim_time = Opm::TimeStampUTC( input.sched.simTime(sim_step + 1) );
```

Comment:
That doesn't look completely right to me.  If there are multiple timesteps within a single report step, then all of those will be attributed to the same day.  That's going to be confusing if the report steps have very long duration (6 months, to take an extreme example).

Reply:
You are absolutely right - thank you!

Reply:
Fixed